app/views/contexts/view_submitted_response_context.py

```diff
@@ -24,22 +25,25 @@ def build_view_submitted_response_context(
     view_submitted_response_expired = has_view_submitted_response_expired(
         questionnaire_store.submitted_at  # type: ignore
     )
+    metadata_proxy = MetadataProxy(questionnaire_store.metadata)
+    trad_as = metadata_proxy.trad_as
+    ru_name = metadata_proxy.ru_name
 
     if survey_type is SurveyType.SOCIAL:
         submitted_text = lazy_gettext("Answers submitted.")
-    elif trad_as := questionnaire_store.metadata.get("trad_as"):
+    elif trad_as and ru_name:
```

Comment:
In the original code we don't check for `ru_name`. `ru_name` is required field now for `business`, `default`,  `northernireland` so we don't need to check for it here?

Reply:
Removed 👍